Frameworks/GLKit/GLKEffect.mm

```diff
@@ -87,7 +87,7 @@ - (BOOL)prepareShaders {
 }
 
 - (void)prepareToDraw {
-    BOOL success = FALSE;
+    bool success = false;
```

Comment:
BOOL is not bool on all platforms.  There are other things that are broken here (such as usage of TRUE/FALSE) instead of (YES/NO), which are rather inconsequential but incorrect.  We should not be changing BOOL to bool in these cases.